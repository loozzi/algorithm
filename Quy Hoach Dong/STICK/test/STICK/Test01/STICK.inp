9

4
97 29
57 40
74 75
44 35

39
9 89
89 21
55 87
4 30
41 2
13 2
54 7
78 65
9 70
40 99
1 87
56 60
90 68
31 99
22 64
84 13
12 21
61 63
24 42
65 2
15 26
46 66
88 97
68 98
100 84
68 84
4 39
99 80
38 9
77 16
55 70
95 82
40 76
93 43
84 1
28 100
12 28
33 21
65 27

26
11 50
37 27
69 77
72 28
82 69
16 34
91 97
58 67
47 80
79 76
36 51
62 47
55 12
9 90
1 68
17 9
2 57
78 55
27 57
60 87
5 13
10 11
22 74
73 57
85 67
4 59

17
8 64
10 74
8 11
11 5
98 16
58 43
35 87
22 77
87 11
9 84
100 14
16 71
21 43
66 24
80 11
55 91
31 2

46
75 21
29 27
66 40
5 37
13 79
41 58
23 60
43 85
51 94
42 15
3 10
89 27
63 64
90 23
48 72
53 76
20 37
91 80
29 44
49 11
39 93
20 66
51 12
81 71
62 56
49 38
52 35
97 73
52 88
76 68
53 52
91 93
33 24
87 33
70 71
71 61
38 85
78 93
19 40
80 71
40 82
28 51
6 51
76 48
16 99
66 77

26
10 88
80 87
77 79
89 68
72 54
58 3
93 74
2 7
67 50
91 20
36 84
15 27
77 81
39 49
7 34
28 64
11 61
56 30
12 23
32 78
42 62
45 91
86 47
19 62
80 22
47 18

31
26 4
84 9
85 15
34 52
13 87
26 28
57 50
95 61
27 12
68 70
41 75
74 80
47 43
96 70
50 91
53 29
22 43
50 26
3 94
12 96
96 21
51 77
92 20
6 90
61 62
52 74
91 16
99 19
7 25
20 33
32 6

18
80 3
20 5
60 69
36 47
79 1
55 92
36 70
18 4
4 66
49 73
90 15
30 31
12 2
18 63
62 27
65 55
25 87
61 92

28
36 30
93 44
44 33
52 91
3 1
25 23
22 49
21 37
4 60
61 59
39 51
8 41
1 24
23 42
62 44
73 42
92 31
21 56
83 6
26 61
79 27
78 28
14 22
39 37
14 86
94 33
74 73
53 50
