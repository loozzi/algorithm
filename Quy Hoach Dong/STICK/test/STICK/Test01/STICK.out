2
9
6
7
10
6
9
5
9
